crates/sui-protocol-config/src/lib.rs

```diff
@@ -13,7 +13,7 @@ use tracing::{info, warn};
 
 /// The minimum and maximum protocol versions supported by this build.
 const MIN_PROTOCOL_VERSION: u64 = 1;
-const MAX_PROTOCOL_VERSION: u64 = 46;
+const MAX_PROTOCOL_VERSION: u64 = 47;
```

Comment:
With the cut tomorrow, was surprised you needed to do this, but makes a bit of sense I suppose looking at what was enabled in 46

Reply:
Yea, was likewise surprised that there wasn't a protocol version sitting already for the cut. But yea, we definitely need to protocol config this change since it will change transaction effects.

Reply:
Of course there is now a merge conflict on another PR that landed that added 47... 🎉